testing/scylla/config/scylla-ssl.yaml

```diff
@@ -1,4 +1,5 @@
 # Scylla SSL enabled configuration.
+# This file merged with testing/config/scylla.yaml during start-dev-env process when SSL_ENABLED=true.
```

Comment:
```suggestion
# This file is merged with testing/config/scylla.yaml during start-dev-env process when SSL_ENABLED=true.
```

Reply:
And overall that's great that you introduce possibility to enable SSL in local env + test it !